pkg/cosign/tlog.go

```diff
@@ -91,24 +98,19 @@ func intotoEntry(ctx context.Context, signature, pubKey []byte) (models.Proposed
 // There are two Env variable that can be used to override this behaviour:
 // SIGSTORE_REKOR_PUBLIC_KEY - If specified, location of the file that contains
 // the Rekor Public Key on local filesystem
-func GetRekorPubs(ctx context.Context, _ *client.Rekor) (map[string]RekorPubKey, error) {
-	publicKeys := make(map[string]RekorPubKey)
+func GetRekorPubs(ctx context.Context) (*TrustedRekorPubKeys, error) {
+	publicKeys := NewTrustedRekorPubKeys()
 	altRekorPub := env.Getenv(env.VariableSigstoreRekorPublicKey)
 
 	if altRekorPub != "" {
 		raw, err := os.ReadFile(altRekorPub)
 		if err != nil {
 			return nil, fmt.Errorf("error reading alternate Rekor public key file: %w", err)
 		}
-		extra, err := PemToECDSAKey(raw)
-		if err != nil {
-			return nil, fmt.Errorf("error converting PEM to ECDSAKey: %w", err)
-		}
-		keyID, err := getLogID(extra)
+		publicKeys.AddRekorPubKey(raw, tuf.Active)
```

Comment:
err = publicKeys.AddRekorPubKey(raw, tuf.Active)

Reply:
done in latest commit w/ the others!